src/pykiso/lib/connectors/cc_serial.py

```diff
@@ -0,0 +1,165 @@
+##########################################################################
```

Comment:
Example yaml and documentation (at least what's new) are missing

Reply:
added docs and loopback example